app/views/users/show.html.haml

```diff
@@ -0,0 +1,34 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %h2 #{@user.full_name}'s video collections (#{@user.queue_items.count})
+          = render 'follow_link', user: @user
+          
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |queue_item|
+              %tr
+                %td= link_to "#{queue_item.video_title}", video_path(queue_item.video)
+                %td= link_to "#{queue_item.category_name}", category_path(queue_item.video_category)
+
+%section.user_reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %header
+        %h3 #{@user.full_name}'s #{@user.reviews_count}
+      %ul
+        - @user.reviews.each do |review|
+          %article.review
+            %li.row
+              .col-sm-2
+                %p
+                  = link_to "\"#{review.video.title}\"", video_path("#{review.video.id}")
```

Comment:
`= link_to review.video.title, review.video` <-- much easier to read. Leave off all the quotes and curly brackets :)